test/MorseL.Client.WebSockets.Tests/MorseL.Client.WebSockets.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>    
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Why is this 2.0? It probably doesn't matter as much now that 2.0 is actually released but we should stick to 1.1 for the next few weeks at least so we don't require tooling upgrades.